spec/features/invalid_session_spec.rb

```diff
@@ -2,16 +2,6 @@
 
 RSpec.describe "Invalid session", :throws_cfe_error, type: :feature do
   it "tells me if CfeService thinks my answers are invalid" do
-    allow(CfeService).to receive(:call).and_raise(Cfe::InvalidSessionError.new(DomesticAbuseApplicantForm.new))
```

Comment:
Just curious, why was this test removed. Doesn't appear to be related to the feature flag stuff.